packages/react/src/create-strict-api/index.ts

```diff
@@ -6,11 +6,11 @@ type PseudosDeclarations = {
   [Q in CSSPseudos]?: StrictCSSProperties;
 };
 
-type EnforceSchema<TObject> = {
-  [P in keyof TObject]?: P extends keyof CompiledSchema
-    ? TObject[P] extends Record<string, unknown>
-      ? EnforceSchema<TObject[P]>
-      : TObject[P]
+type EnforceSchema<TSchema> = {
+  [P in keyof TSchema]?: P extends keyof CompiledSchema
+    ? TSchema[P] extends Record<string, any>
```

Comment:
Main fix: `unknown` -> `any`

Reply:
Interesting, do you know why this works? I've seen similar things, just never seen like a TS issue or explanation.

Reply:
I think it's an edge case with complex nested types (interface + omits + ?). At a glace unknown should be fine but in this case it's obviously not.

I'm not sure!